molecule/Dockerfile-debian-java8.j2

```diff
@@ -18,12 +18,17 @@ RUN apt-get update && \
 RUN /usr/bin/printf '\xfe\xed\xfe\xed\x00\x00\x00\x02\x00\x00\x00\x00\xe2\x68\x6e\x45\xfb\x43\xdf\xa4\xd9\x92\xdd\x41\xce\xb6\xb2\x1c\x63\x30\xd7\x92' > /etc/ssl/certs/java/cacerts && \
     /var/lib/dpkg/info/ca-certificates-java.postinst configure
 
-RUN wget -qO - https://packages.confluent.io/clients/deb/archive.key | sudo apt-key add - && \
-    add-apt-repository "deb https://packages.confluent.io/clients/deb/ stretch stable main" && \
+{% set PACKAGE_VER = lookup('env', 'VERSION') | default('7.0.1', true) %}
```

Comment:
I see these changes are expected in all the branches. Probably you need to update again for patches. Can we not have a simple sed/awk command to parse it from variables. 
Sorry for overlooking this earlier.

Reply:
From which variables, @nsharma-git ?
In all jenkins jobs, I'll make an update to read these variables from molecule args
For ex: 
When someone use `molecule .. --e --e "confluent_common_repository_baseurl=xyz confluent_client_repository_baseurl=abc confluent_package_version=7.1.0"` in the jenkins jobs, a script will resolve these variables COMMON_REPO_URL, VERSION, CLIENT_REPO_URL from here.

Reply:
Can we not parse the version from https://github.com/confluentinc/cp-ansible/blob/353ddc442e4eda5b023a9f257a571a01a0fd76e4/roles/variables/defaults/main.yml#L5 in default case ?

Reply:
@nsharma-git 
Please have a look. Only when this looks fine, I'll do the change for other PRs.